app/state/reducers/searchFiltersReducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = {
+  search: '',
+};
+
+function searchFiltersReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'ENTER_OR_CHANGE_SEARCH_PAGE': {
```

Comment:
Now that I think about it, didn't we talk about starting to use redux-actions' handleActions?

```js
export default handleActions({
  ENTER_OR_CHANGE_SEARCH_PAGE: (state, action) => ({ ...initialState, ...action.payload }),
}, initialState);
```